taipy/core/_orchestrator/_orchestrator.py

```diff
@@ -39,7 +39,6 @@ class _Orchestrator(_AbstractOrchestrator):
     blocked_jobs: List = []
```

Comment:
You removed `_submissition_entities` and that's great!
But why not `blocked_jobs` ? These are also a state that we might want to get rid of.

Reply:
Yep I will :D just that we don't have a good strategy for that atm. For Submission, we already have the repo concept to work with. But for orchestrator, we don't have anything like that. For remote execution with rabbitmq, I think we can take advantage of persistency provided by rabbitmq for this as a simple solution.